src/HexBug/cogs/info.py

```diff
@@ -107,6 +107,12 @@ class InfoMessages(Enum):
         content="https://media.hexxy.media/data/great_spells.zip"
     )
 
+    gtp_itemdrop = MessageProps(
+        content="""When someone casts Greater Teleport on themself, each item in their inventory (except for those in their hands and any slots added by mods) has a random chance to drop at their destination.
+        For items in the main three inventory rows, the chance is N/10,000. For items in their hotbar, the chance is N/20,000. For items in their armor slots, the chance is N/40,000.
```

Comment:
This needs to be dedented (see `bug_report`) - otherwise the indentation will show up in the actual string.